lib/dirac_clover.cpp

```diff
@@ -92,6 +92,11 @@ namespace quda {
     CoarseOp(Y, X, T, *gauge, &clover, kappa, a, mu_factor, QUDA_CLOVER_DIRAC, QUDA_MATPC_INVALID);
   }
 
+  void DiracClover::prefetch(QudaFieldLocation mem_space, cudaStream_t stream) const {
+    Dirac::prefetch(mem_space, stream);
+    clover.prefetch(mem_space, stream);
```

Comment:
Perhaps should specialize for preconditioned operator.  `DiracClover` should only fetch clover field, and `DiracCloverPC` operator should grab the direct and inverse field according to the type of preconditioning, e.g., both parities of inverse for symmetric preconditioned operator or one parity of each for asymmetric operator.